packages/wdio-cli/src/templates/wdio.conf.tpl.ejs

```diff
@@ -85,6 +85,14 @@ exports.config = {
         maxInstances: 5,
         //
         browserName: 'firefox'
+    // }, {
+        // If outputDir is provided WebdriverIO can capture driver session logs
+        // it is possible to configure which logTypes to include/exclude.
+        // excludeDriverLogs: ['*'], // pass '*' to exclude all driver session logs
+        // excludeDriverLogs: ['bugreport', 'server'],
+        // "appium:deviceName": "Pixel 2",
+        // "appium:platformName": "Android",
+        // "appium:platformVersion": "9"
```

Comment:
same as above .. let's just incorporate these

Reply:
Do you want to set logLevel of wdio loggers (not driver logs) in capabilities?

Can't imagine why would we need to set wdio loggers per capability.